### ISSUE TITLE
Lint/SymbolConversion fails if to_sym called on implicit receiver

### ISSUE THREAD

**Issue by zhuravel** [rubocop-hq/rubocop#9463]
Sample code that generates errors:
```ruby
class ViewSetup
  def offer_share_view?
    to_sym == SHARE_IDENTIFIER
  end

  def to_sym
    view_category.to_sym
  end

  def css_class
    case to_sym
    when SIGNUP_IDENTIFIER then 'signup'
    when SHARE_IDENTIFIER then 'share'
    end
  end
end
```

## RuboCop version

```
$ rubocop -V
1.9.0 (using Parser 3.0.0.0, rubocop-ast 1.4.1, running on ruby 2.7.2 x86_64-darwin19)
  - rubocop-performance 1.9.2
  - rubocop-rails 2.9.1
  - rubocop-rake 0.5.1
  - rubocop-rspec 2.1.0
```

**Comment by koic** [comment]
This issue is a dup with #9442 and resolved by #9440.

**Comment by dvandersluis** [comment]
Thanks for the report, but this is a duplicate of #9440, and already fixed by #9442.
